bench/lean4/optimized/CompletionName.ll

```diff
@@ -176,8 +175,12 @@ l___private_Lean_Meta_CompletionName_0__Lean_Meta_isInternalNameModuloPrivate.ex
   br label %lean_dec.exit
 
 lean_dec.exit:                                    ; preds = %30, %29, %27, %l___private_Lean_Meta_CompletionName_0__Lean_Meta_isInternalNameModuloPrivate.exit
-  %31 = inttoptr i64 %.1.i to ptr
-  ret ptr %31
+  %31 = shl nuw nsw i32 %.0.i.i, 1
+  %32 = and i32 %31, 510
+  %33 = or disjoint i32 %32, 1
+  %34 = zext nneg i32 %33 to i64
+  %35 = inttoptr i64 %34 to ptr
```

Comment:
Regression: The problem here is basically that for a phi node with constant arguments, we can fold following operations into it. After it has been converted into an instruction, this is no longer possible.